spec/spec/spec_helper.rb

```diff
@@ -0,0 +1,89 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
probably popped that extra one on there when I was getting rspec up and running. Looks like you cleared it now.